clouddriver-integration/src/test/java/com/netflix/spinnaker/clouddriver/BaseContainerTest.java

```diff
@@ -5,110 +5,66 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
+ *   http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.netflix.spinnaker.clouddriver;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import java.net.URI;
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.time.Duration;
-import java.util.Map;
-import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInfo;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.Network;
-import org.testcontainers.containers.output.Slf4jLogConsumer;
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.junit.jupiter.Testcontainers;
 import org.testcontainers.utility.DockerImageName;
 
 @Testcontainers
-class StandaloneContainerTest {
-
-  private static final String REDIS_NETWORK_ALIAS = "redisHost";
-
-  private static final int REDIS_PORT = 6379;
+public class BaseContainerTest {
 
-  private static final Logger logger = LoggerFactory.getLogger(StandaloneContainerTest.class);
+  private static final Logger logger = LoggerFactory.getLogger(BaseContainerTest.class);
 
-  private static final Network network = Network.newNetwork();
+  protected static final Network network = Network.newNetwork();
 
-  private static final GenericContainer redis =
-      new GenericContainer(DockerImageName.parse("library/redis:5-alpine"))
-          .withNetwork(network)
-          .withNetworkAliases(REDIS_NETWORK_ALIAS)
-          .withExposedPorts(REDIS_PORT);
-
-  private static GenericContainer clouddriverContainer;
+  protected static GenericContainer clouddriverContainer;
```

Comment:
Is it really correct to have this be static?  It does seem to work, but I can't tell if we're getting lucky.  Don't we want separate clouddriver containers for each test class?  And probably the network too.

Reply:
The way I separated the configs between base and sub classes will make the tests always work but I see it doesn't seem right to code it this way. I will refactor this.

Reply:
Updated all the classes now. The test setup and containers are fully isolated now.

Reply:
Thanks!